src/json_interface.jl

```diff
@@ -101,32 +102,62 @@ save model constructor, model parameters, and other information to a JSON file.
 * `errors` : the fitting / test errors computed during the fitting 
 * `verbose` : print information about the saving process     
 """
-function save_model(model, filename; 
-                    make_model_args = nothing, 
-                    errors = nothing, 
-                    verbose = true, 
-                    meta = Dict(), )
-
-   D = Dict("model_parameters" => model.ps, 
-            "meta" => meta)
-
-   if isnothing(make_model_args) 
-      if verbose
-         @warn("Only model parameters are saved but no information to reconstruct the model.")
-      end
-   else 
-      D["make_model_args"] = make_model_args
-   end
 
-   if !isnothing(errors)
-      D["errors"] = errors
-   end
 
-   open(filename, "w") do io
-       write(io, JSON.json(D, 3))
-   end
+function find_manifest(start_path = pwd())
```

Comment:
This doesn't look right. I think it should be read from the current project. Something like this: 
```julia-repl
shell> pwd
/Users/ortner/gits/ACEpotentials.jl/docs/src/tutorials

julia> using Pkg

julia> p = Pkg.project()
Pkg.API.ProjectInfo("ACEpotentials", UUID("3b96b61c-0fcc-4693-95ed-1ef9f35fcc53"), v"0.8.0-dev", true, Dict{String, Base.UUID}("Pkg" => UUID("44cfe95a-1eb2-52ea-b672-e2afdf69b78f"), "ForwardDiff" => UUID("f6369f11-7733-5829-9624-2563aa707210"), "SpheriCart" => UUID("5caf2b29-02d9-47a3-9434-5931c85ba645"), "Random" => UUID("9a3f8284-a2c9-5f02-9a11-845980a1fd5c"), "Polynomials4ML" => UUID("03c4bcba-a943-47e9-bfa1-b1661fc2974f"), "RepLieGroups" => UUID("f07d36f2-91c4-427a-b67b-965fe5ebe1d2"), "Interpolations" => UUID("a98d9a8b-a2ab-59e6-89dd-64a1c18fca59"), "ACEfit" => UUID("ad31a8ef-59f5-4a01-b543-a85c2f73e95c"), "AtomsCalculators" => UUID("a3e0e189-c65a-42c1-833c-339540406eb1"), "Optim" => UUID("429524aa-4258-5aef-a3af-852621145aeb")…), "/Users/ortner/gits/ACEpotentials.jl/Project.toml")

julia> p.
dependencies  ispackage
name          path
uuid          version
julia> p.path
"/Users/ortner/gits/ACEpotentials.jl/Project.toml"
```

Reply:
Done